src/services/logout-service.ts

```diff
@@ -1,65 +1,65 @@
 import * as crypto from 'crypto'
 import chalk from 'chalk'
-import axios, {AxiosRequestConfig} from 'axios'
-import {vars} from '../vars'
-import {CliUx} from '@oclif/core'
-import {LogService} from '../services/log-service'
-import {AbortService} from '../services/abort-service'
+import axios, { AxiosRequestConfig } from 'axios'
+import { vars } from '../vars'
+import { CliUx } from '@oclif/core'
+import { LogService } from '../services/log-service'
+import { AbortService } from '../services/abort-service'
 
 interface LogoutServiceAttrs {
-  cmd;
-  yes;
+  cmd: any
+  yes: boolean
 }
 
 class LogoutService {
-  public cmd;
-  public yes;
-  public log;
-  public requestUid;
-  public controller;
-  public abort;
-  public checkCount;
+  public cmd: any
+  public yes: boolean
+  public log: LogService
+  public abort: AbortService
+  public requestUid: string
+  public controller?: AbortController
+  public checkCount = 0
+  public MAX_CHECK_COUNT = 50
 
   constructor(attrs: LogoutServiceAttrs = {} as LogoutServiceAttrs) {
     this.cmd = attrs.cmd
     this.yes = attrs.yes
-    this.log = new LogService({cmd: attrs.cmd})
-    this.abort = new AbortService({cmd: attrs.cmd})
-
+    this.log = new LogService({ cmd: attrs.cmd })
+    this.abort = new AbortService({ cmd: attrs.cmd })
     const rand = crypto.randomBytes(32).toString('hex')
     this.requestUid = `req_${rand}`
-    this.checkCount = 0
   }
 
   async run(): Promise<void> {
     await this.logout()
   }
 
-  async logout(tip = true): Promise<void> {
+  async logout(showInstructions = true): Promise<void> {
     if (!this.yes) {
       this.log.local(`Logout URL: ${this.logoutUrl}`)
       const answer = await CliUx.ux.prompt(`${chalk.dim(this.log.pretextLocal)}Press ${chalk.green('y')} (or any key) to logout and revoke credential (.env.me) or ${chalk.yellow('q')} to exit`)
-      if (answer === 'q' || answer === 'Q') {
+      if (answer.toLowerCase() === 'q') {
```

Comment:
can we have this also strip any spaces? - in case the user accidentally hits the space bar and then `q`